packages/dashboard-backend/jest.config.js

```diff
@@ -19,10 +19,11 @@ module.exports = {
   moduleDirectories: [
     'node_modules',
   ],
-  collectCoverageFrom: [
-    'src/api/**/*.ts',
-    'src/services/**/*.ts',
-    'src/devworkspace-client/*.ts',
-    'src/devworkspace-client/services/**/*.ts',
+  coveragePathIgnorePatterns: [
+    'src/devworkspaceClient/__tests__/*',
   ],
+  testPathIgnorePatterns: [
+    'src/devworkspaceClient/__tests__/errors.spec.ts',
+    'src/devworkspaceClient/__tests__/integration.spec.ts',
```

Comment:
@akurinnoy I think we should remove  'src/devworkspaceClient/__tests__/errors.spec.ts' and
    'src/devworkspaceClient/__tests__/integration.spec.ts' at all.